Article/Article.js

```diff
@@ -5,28 +5,34 @@ class Article {
     // assign this.element to the passed in article element
     this.element = element;
     // create a reference to the ".expandButton" class. 
-    this.expandButton = document.querySelector('.expandButton');
+
+    
+    this.expandButton = document.getElementsByClassName('expandButton');
+    
 
 
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    this.expandButton.innerHTML = 'expand';
+    for(let i = 0; i < this.expandButton.length; i++){
```

Comment:
I would suggest you use some of the es6 methods to accomplish this, just to get used to them.